:memo: Update the getting help page (#1963)
platformsh/platformsh-docs: docs/src/overview/get-support.md

---
title: " Get support"
weight: 4
toc: false
description: |

>   Find out how to get help If you're experiencing an issue with Platform.sh.

Reviewer: ```suggestion
  Find out how to get help if you're experiencing issues with Platform.sh.
```

---

{{< description >}}

> ## Create a support ticket

Reviewer: As with the items below, this should include a description of when it should be chosen. Not all issues should be support tickets. For example, support doesn't really cover issues with app development. And what they do cover and how fast might be influenced by the plan. I don't think you have to get into all those details, but at least something would be helpful.

Author: I've added a description as per M.Antikaapo instructions.

> Create a support ticket when you're experiencing issues related to the proper functioning of the Platform.sh infrastructure, application container software, build processes, possible bugs or if you have general questions.

Reviewer: ```suggestion
If you're experiencing issues related to the proper functioning of the Platform.sh infrastructure, application container software, or build processes; have found possible bugs; or have general questions, open a support ticket:
```

2. Select **User menu** (your name) > **Support**.
3. Click **+ New ticket**.
4. Fill in the ticket fields and click **Submit**.

> In the future, you can [access the ticket list](https://console.platform.sh/-/users/~/tickets) or [open a ticket directly](https://console.platform.sh/-/users/~/tickets/open) using the shortcuts.

Reviewer: ```suggestion
Or use these shortcuts to [access all support tickets](https://console.platform.sh/-/users/~/tickets) or [open a new ticket](https://console.platform.sh/-/users/~/tickets/open).
```

> Once you submit a ticket, you can see it on the list alongside tickets created by users in your organization over time.

Reviewer: ```suggestion
Once you submit a ticket, you see it in a list of all tickets created within your organization.
```
The list wasn't described before, so I wouldn't use "the".

> Note that once you submit the ticket you won't be able to modify or delete the submission.

Reviewer: ```suggestion
Note that once you submit the ticket, you won't be able to modify or delete the submission.
```

If you have any additional information, you can select the submitted ticket and write a message.

## Slack

> If you want to chat with engineers and other customers you can join the [public Slack channel](https://chat.platform.sh/).

Reviewer: ```suggestion
To chat with engineers and other customers, join the [public Slack channel](https://chat.platform.sh/).
```
Again, I think chatting doesn't match the experience the user is probably currently having. It feels too positive and not focused on a resolution. What kinds of questions should they bring there? Or when to go there? The idea is that there should be some sort a decision tree for deciding which channel to take when I have an issue. So something that coordinates with the reasoning for a support ticket.

Author: As I mentioned above based on a conversation with Support. They say:
"That is a great question! And I don’t think there is a well defined answer to it.
Support team does monitor the channel to make sure any problems customers might bring up there are addressed. But at the same time. we don’t really like to turn it into a support ticket escalation channel, as there’s no SLA for Pro support requests.
Lot of the others who answer more application or framework related questions do so voluntarily based on their own expertise. Not sure we’ve ever agreed on the real goal of the channel, some hate it, some think it’s a necessary evil, but I don’t think anyone considers it an ideal solution for community based knowledge sharing or help."

> If you want to talk to with engineers and other customers you can join the [public Slack channel](https://chat.platform.sh/).

Reviewer: ```suggestion
To talk with engineers and other customers, join the [public Slack channel](https://chat.platform.sh/).
```

I saw the message from Support both times you posted it. But I think people are coming to this page looking for support, not Support :wink: 

What I think that means is they still want to know why they would want to join here and what kinds of questions might be answered. I might also place the "other customers" first to emphasize that it's not a great way to get immediate contact with Platform engineers.

But mostly, I'd want to emphasize that it's better for questions about app development or whatever that don't fit above but without any expectation that they will necessarily get a reply.

> In the [Platform.sh Community Site](https://community.platform.sh/) you can find plenty of how-to guides that contains suggestions on how to make the most out of Platform.

Reviewer: ```suggestion
The [Platform.sh Community site](https://community.platform.sh/) has how-to guides with suggestions on how to get the most out of Platform.sh.
```
I might also add that they can add additional questions, but it's not a blocking issue.

## Contact Sales

> If you have questions with regards pricing or if you need help figuring out if Platform.sh is the right solution for your team, get in touch with [Sales](https://platform.sh/contact/).

Reviewer: ```suggestion
If you have questions about pricing or need help figuring out if Platform.sh is the right solution for your team, get in touch with [Sales](https://platform.sh/contact/).
```

It could be "questions with regards to" or "questions regarding", but they're both too formal (especially the first) and not as easily understood.

> ---

Reviewer: ```suggestion
```

---
title: " Get support"
weight: 4
toc: false
description: |

>   If you're experiencing a problem with Platform.sh, you can submit a support ticket, join the Slack channel, or checkout the Platform.sh Community.

Author: I'm happy to use "issue" from now on.

---

{{< description >}}

## Create a support ticket

To create a ticket from the console, follow these steps:

Before you begin

* You need to have a Platform.sh account.
* You must be logged into your account.

Steps

1. [Log in to Platform.sh](https://auth.api.platform.sh/)
2. In the console, click your **user profile** > **Support**.

Now you are on the **Support tickets** page.
Here you can see the list of tickets created by users in your organization over time.
If the page has no tickets, it means you're the first user in your organization creating one.

3. Click **+ New ticket**.
4. Fill in the ticket fields and click **Submit**.

Note that once you submit the ticket you won't be able to modify or delete the submission.
If you have any additional information you can add it to the ticket as a reply.

## Other ways to get support

### Slack

Hang out with customers and engineers on the [public Slack channel](https://chat.platform.sh/)

> Mention `@platform` to get the team attention.

Author: Thanks, The group and slack channel are mentioned on the main marketing site. I'll let Marketing know.

> Mention `@platform` to get the team attention.

Author: I've spoken with Support and they literally say: "Support team does monitor the channel to make sure any problems customers might bring up there are addressed. But at the same time. we don’t really like to turn it into a support ticket escalation channel, as there’s no SLA for Pro support requests.
Lot of the others who answer more application or framework related questions do so voluntarily based on their own expertise"

I've rephrased the info under Slack but I wouldn't give too many details tbh.
